<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>923</width>
    <height>662</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="5,1">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="image_viewer">
       <property name="minimumSize">
        <size>
         <width>410</width>
         <height>0</height>
        </size>
       </property>
       <property name="title">
        <string>Image Viewer</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="GBOX_alarm">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>60</height>
        </size>
       </property>
       <property name="title">
        <string>Notification</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="NON_mail">
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>E-mail:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="TXT_mail">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="CHBOX_mail">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,4,3,0,0,0">
     <property name="spacing">
      <number>6</number>
     </property>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="maximumSize">
        <size>
         <width>410</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title">
        <string>Scan Settings</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <item>
         <layout class="QGridLayout" name="gridLayout_3" rowstretch="1,0,0">
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="frameShape">
             <enum>QFrame::Panel</enum>
            </property>
            <property name="text">
             <string>X</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QDoubleSpinBox" name="GUI_y_start">
            <property name="decimals">
             <number>3</number>
            </property>
            <property name="maximum">
             <double>12.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_3">
            <property name="frameShape">
             <enum>QFrame::Panel</enum>
            </property>
            <property name="text">
             <string>Y</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QDoubleSpinBox" name="GUI_x_step">
            <property name="decimals">
             <number>3</number>
            </property>
            <property name="maximum">
             <double>12.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
            <property name="value">
             <double>0.100000000000000</double>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QDoubleSpinBox" name="GUI_y_step">
            <property name="decimals">
             <number>3</number>
            </property>
            <property name="maximum">
             <double>12.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
            <property name="value">
             <double>0.100000000000000</double>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QLabel" name="label_6">
            <property name="frameShape">
             <enum>QFrame::Panel</enum>
            </property>
            <property name="text">
             <string>Step</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QDoubleSpinBox" name="GUI_y_stop">
            <property name="decimals">
             <number>3</number>
            </property>
            <property name="maximum">
             <double>12.000000000000000</double>
            </property>
            <property name="value">
             <double>0.500000000000000</double>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_5">
            <property name="frameShape">
             <enum>QFrame::Panel</enum>
            </property>
            <property name="text">
             <string>Stop</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label">
            <property name="frameShape">
             <enum>QFrame::Panel</enum>
            </property>
            <property name="text">
             <string>Start</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QDoubleSpinBox" name="GUI_x_stop">
            <property name="decimals">
             <number>3</number>
            </property>
            <property name="maximum">
             <double>12.000000000000000</double>
            </property>
            <property name="value">
             <double>0.500000000000000</double>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_4">
            <property name="frameShape">
             <enum>QFrame::Panel</enum>
            </property>
            <property name="text">
             <string>mm</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="GUI_x_start">
            <property name="decimals">
             <number>3</number>
            </property>
            <property name="maximum">
             <double>12.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QFormLayout" name="formLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>PMT Exposure Time (ms)</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="LE_pmt_exposure_time_in_ms">
            <property name="text">
             <string>1</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string>PMT Average Number (#)</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="LE_pmt_average_number">
            <property name="text">
             <string>1</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QCheckBox" name="CB_auto_go_to_max">
            <property name="text">
             <string>Go to the max position after the scanning is finished</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QPushButton" name="BTN_scan_vicinity">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>25</height>
             </size>
            </property>
            <property name="sizeIncrement">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Arial Black</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Scan Around the Current Position</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="SPBOX_vicinity_length">
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="minimum">
             <number>2</number>
            </property>
            <property name="maximum">
             <number>5</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="maximumSize">
        <size>
         <width>410</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title">
        <string>Scan Progress</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_8" stretch="6,1">
        <property name="spacing">
         <number>6</number>
        </property>
        <item>
         <layout class="QFormLayout" name="formLayout">
          <property name="labelAlignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="formAlignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="verticalSpacing">
           <number>6</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label_7">
            <property name="font">
             <font>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>Latest Count</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="LBL_latest_count">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>15</height>
             </size>
            </property>
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_8">
            <property name="font">
             <font>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>Points Done</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="LBL_points_done">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>15</height>
             </size>
            </property>
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_9">
            <property name="font">
             <font>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>Total Points</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="LBL_total_points">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>15</height>
             </size>
            </property>
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_13">
            <property name="font">
             <font>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>Save File</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="LE_save_file_name"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_14">
            <property name="font">
             <font>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>Save Dir</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="LE_save_file_dir"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="sizeConstraint">
           <enum>QLayout::SetMinimumSize</enum>
          </property>
          <item>
           <widget class="QPushButton" name="BTN_select_save_dir">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>25</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Arial Black</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Change Dir</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="BTN_save">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>25</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Arial Black</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Save</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="BTN_load_save_file">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>25</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Arial Black</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Load</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="maximumSize">
        <size>
         <width>410</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title">
        <string>Plot Settings</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QFormLayout" name="formLayout_2">
            <item row="0" column="0">
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>Min</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>Max</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="plot_min">
              <property name="text">
               <string>0</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="plot_max">
              <property name="text">
               <string>100</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QCheckBox" name="CB_auto_minmax">
              <property name="text">
               <string>Auto MinMax</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="CB_flip_vertically">
              <property name="text">
               <string>Flip Vertically</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="CB_flip_horizontally">
              <property name="text">
               <string>Flip Horizontally</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="BTN_apply_plot_settings">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>25</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Arial Black</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Apply</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BTN_pause_or_resume_scanning">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>410</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Arial Black</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Pause Scanning</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BTN_start_scanning">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>410</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Arial Black</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Start Scanning</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BTN_go_to_max">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>410</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Arial Black</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Go to Max</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>BTN_start_scanning</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>pressedScanStartButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>901</x>
     <y>613</y>
    </hint>
    <hint type="destinationlabel">
     <x>911</x>
     <y>515</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BTN_select_save_dir</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>pressedChangeSaveDir()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>670</x>
     <y>387</y>
    </hint>
    <hint type="destinationlabel">
     <x>911</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BTN_pause_or_resume_scanning</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>pressedPauseScanning()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>901</x>
     <y>577</y>
    </hint>
    <hint type="destinationlabel">
     <x>911</x>
     <y>468</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BTN_scan_vicinity</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>pressedScanVicinityButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>812</x>
     <y>205</y>
    </hint>
    <hint type="destinationlabel">
     <x>911</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BTN_apply_plot_settings</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>pressedApplyButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>891</x>
     <y>531</y>
    </hint>
    <hint type="destinationlabel">
     <x>909</x>
     <y>449</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BTN_go_to_max</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>pressedGoToMax()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>901</x>
     <y>649</y>
    </hint>
    <hint type="destinationlabel">
     <x>911</x>
     <y>528</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>GUI_x_step</sender>
   <signal>valueChanged(double)</signal>
   <receiver>Form</receiver>
   <slot>changedStepValue()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>890</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>911</x>
     <y>62</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>GUI_y_step</sender>
   <signal>valueChanged(double)</signal>
   <receiver>Form</receiver>
   <slot>changedStepValue()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>890</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>911</x>
     <y>207</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BTN_load_save_file</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>pressedLoadSaveFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>890</x>
     <y>387</y>
    </hint>
    <hint type="destinationlabel">
     <x>911</x>
     <y>350</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BTN_save</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>pressedSaveButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>762</x>
     <y>369</y>
    </hint>
    <hint type="destinationlabel">
     <x>911</x>
     <y>410</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>pressedScanStartButton()</slot>
  <slot>goToMax()</slot>
  <slot>pressedApplyButton()</slot>
  <slot>pressedScanVicinityButton()</slot>
  <slot>pressedGoToMax()</slot>
  <slot>changedStepValue()</slot>
  <slot>pressedLoadSaveFile()</slot>
  <slot>pressedChangeSaveDir()</slot>
  <slot>pressedPauseScanning()</slot>
  <slot>pressedSaveButton()</slot>
 </slots>
</ui>
